internal/charmstore/common_test.go

```diff
@@ -23,6 +25,43 @@ type commonSuite struct {
 func (s *commonSuite) addRequiredCharms(c *gc.C, bundle charm.Bundle) {
 	store := s.newStore(c, true)
 	defer store.Close()
+	addRequiredCharms(c, store, bundle)
+}
+
+func (s *commonSuite) newStore(c *gc.C, withES bool) *Store {
```

Comment:
I presume withES means with ElasticSearch, but it would be more clear to me if you spelled out elastic search.